scripts/code-rabbit-yaml.ts

```
@@ -0,0 +1,66 @@
+        .map((line) => {
+            if (line.trim().startsWith('-') || line.trim().startsWith('```')) {
+                return `- ${line.trim().replace(/^- /, '')}`;
+            }
+            if (line.trim() === '') return '';
+            return `  ${line.trim()}`;
+        })
+        .join('\n');
+}
+
+function appendInstructions(templateYaml: string, additions: Record<string, string>): string {
+    const doc = yaml.load(templateYaml) as any;
```

ESLint

[error] Avoid casting objects. Add necessary type checks instead.

(@nangohq/custom-integrations-linting/no-object-casting)